service-workers/cache-storage/script-tests/cache-match.js

```diff
@@ -394,13 +394,12 @@ cache_test(async (cache) => {
     assert_true(original_response_type.includes(original_type),
                 'original response should include the expected mime type');
 
-    // Verify overwriting the content-type header does not change the mime
-    // type.  It should be fixed at Response construction time.
+    // Verify overwriting the content-type header changes the mime type.
     const overwritten_response = new Response('hello world', init_with_headers);
     overwritten_response.headers.set('content-type', 'text/plain');
     const overwritten_response_type = (await overwritten_response.blob()).type;
-    assert_equals(overwritten_response_type, original_response_type,
-                  'original and overwritten response mime types should match');
+    assert_equals(overwritten_response_type, 'text/plain',
```

Comment:
nit:  It might be nice to pull `text/plain` out into a constant that is referenced multiple times to make it clear that they should be identical.